index.js

```diff
@@ -0,0 +1,4 @@
+module.exports.Docker = require('./lib/docker')
+module.exports.Swarm = require('./lib/swarm')
+// for testing
+module.exports._BaseClient = require('./lib/base')
```

Comment:
Unfortunately I need to expose it here in order to stub functions on the base class.
I can't stub functions on the Docker or Swarm instance. I use this `_BaseClient` in other places where library is used